pkg/metrics/metrics.go

```diff
@@ -206,57 +212,74 @@ var (
 		Help:      "Number of errors that occurred in the datapath or datapath management",
 	},
 		[]string{LabelDatapathArea, LabelDatapathName, LabelDatapathFamily})
+
+	// Health statistics
+
+	// Latency is a histogram showing latency to nodes
+	Latency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "latency",
```

Comment:
Please set this up, curl it and pass the results into `promtool check-metrics`. This will provide suggestions about naming.

At minimum it should state the units, and probably whether it's a sum or count.

If you haven't read up on the following pages, they may prove interesting for how to choose names and types for prometheus metrics:
https://prometheus.io/docs/practices/naming/
https://prometheus.io/docs/practices/histograms/

Reply:
Thanks for the hint. I added `_seconds` suffix to the name and it's accepted by promtool.